.github/workflows/ci.yaml

```diff
@@ -501,6 +501,7 @@ jobs:
           publish_dir: docs/_site
           external_repository: lampepfl/dotty-website
           publish_branch: gh-pages
+          keep_files: true
```

Comment:
I believe this line will preserve older static site files while publishing nightlies according to:
https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-keeping-existing-files-keep_files

Reply:
Actually this could not work (would overwrite top level index.html), so I updated `genDocs` script